src/integrationTest/java/uk/gov/hmcts/reform/cpo/repository/PostgresqlContainer.java

```diff
@@ -0,0 +1,34 @@
+package uk.gov.hmcts.reform.cpo.repository;
+
+import org.testcontainers.containers.PostgreSQLContainer;
+
+@SuppressWarnings("PMD.AvoidSynchronizedAtMethodLevel")
+public final class PostgresqlContainer extends PostgreSQLContainer<PostgresqlContainer> {
```

Comment:
This whole class can be deleted - through the existing magic of what's currently in `master`, a postgres container will already start up and be automatically configured as the datasource

Reply:
👍

Reply:
Fixed in https://github.com/hmcts/cpo-case-payment-orders-api/pull/29/commits/997e6c96e81b2bb04894eeb6e64f23be1a43b5c9

Reply:
The magic is in the `application-itest.yaml` file, specifically the `datasource.url`

 `jdbc:tc:postgresql:11:///databasename`
 
 Note there is no `host` or `port` specified - these are not needed and the `testcontainers` library with automagically creatae an instance of the container
 
 More details [heee at testcontainters.org](https://www.testcontainers.org/modules/databases/jdbc/)